src/app/payment/components/checkout/ProductCardComponent.tsx

```diff
@@ -1,65 +1,119 @@
 import { Check, SealPercent, X } from '@phosphor-icons/react';
-import { bytesToString } from 'app/drive/services/size.service';
-import { useTranslationContext } from 'app/i18n/provider/TranslationProvider';
-import { UpsellManagerProps } from 'app/payment/views/IntegratedCheckoutView/CheckoutViewWrapper';
-import TextInput from 'app/share/components/ShareItemDialog/components/TextInput';
-import Button from 'app/shared/components/Button/Button';
-import { useThemeContext } from 'app/theme/ThemeProvider';
+
+import { bytesToString } from '../../../drive/services/size.service';
+import { useTranslationContext } from '../../../i18n/provider/TranslationProvider';
+import { UpsellManagerProps } from '../../../payment/views/IntegratedCheckoutView/CheckoutViewWrapper';
+import TextInput from '../../../share/components/ShareItemDialog/components/TextInput';
+import Button from '../../../shared/components/Button/Button';
+import { useThemeContext } from '../../../theme/ThemeProvider';
 import { ReactComponent as GuaranteeDarkDays } from 'assets/icons/checkout/guarantee-dark.svg';
 import { ReactComponent as GuaranteeWhiteDays } from 'assets/icons/checkout/guarantee-white.svg';
-import { useState } from 'react';
-import { CouponCodeData, Currency, CurrentPlanSelected } from '../../types';
+import { CouponCodeData, Currency, RequestedPlanData } from '../../types';
+import { SelectUsersComponent } from './SelectUsersComponent';
 
 interface ProductFeaturesComponentProps {
-  selectedPlan: CurrentPlanSelected;
-  couponCodeData?: CouponCodeData;
-  couponError?: string;
+  selectedPlan: RequestedPlanData;
+  seatsForBusinessSubscription: number;
   upsellManager: UpsellManagerProps;
+  onUsersChange: (users: number) => void;
   onRemoveAppliedCouponCode: () => void;
   onCouponInputChange: (promoCode: string) => void;
+  couponCodeData?: CouponCodeData;
+  couponError?: string;
 }
 
 const Separator = () => <div className="border border-gray-10" />;
 
-export const getProductAmount = (amount: DisplayPrice['amount'], couponCodeData?: CouponCodeData): number => {
+export const getProductAmount = (
+  amount: DisplayPrice['amount'],
+  users: number,
+  couponCodeData?: CouponCodeData,
+): number => {
   if (couponCodeData?.amountOff) {
-    return amount - couponCodeData.amountOff / 100;
+    return (amount - couponCodeData.amountOff / 100) * users;
   }
 
   if (couponCodeData?.percentOff) {
     const discount = 100 - couponCodeData.percentOff;
 
-    return (amount * discount) / 100;
+    return ((amount * discount) / 100) * users;
   }
 
-  return amount;
+  return amount * users;
+};
+
+const getTextContent = (
+  users: number,
+  isBusiness: boolean,
+  bytes: string,
+  selectedPlan: RequestedPlanData,
+  translate: (key: string, props?: Record<string, unknown>) => string,
+  translateList: (key: string, props?: Record<string, unknown>) => string[],
+) => {
+  const maxUploadGBfile = bytes === '1TB' ? '5GB' : '20GB';
```

Comment:
Extract to a const that give more context about this conditional. 
For example, you could add a const that gives you the info that if it is a standard bussines plan,
```
const STANDARD_BUSINESS_PLAN_SPACE = '1TB'
const FILE_SIZE_LIMIT_STANDARD_BUSINESS_PLAN = '5TB';
const FILE_SIZE_LIMIT_PRO_BUSINESS_PLAN = '20TB';
const isStandardBusinessPlan = bytes ===  STANDARD_BUSINESS_PLAN_SPACE ? FILE_SIZE_LIMIT_STANDARD_BUSINESS_PLAN : FILE_SIZE_LIMIT_PRO_BUSINESS_PLAN;
```
In this way, the reasoning can be better understood just by reading the code